hatch_build.py

```
@@ -0,0 +1,116 @@
+                # Sort to get the highest version number
+                matches.sort(reverse=True)
+                return matches[0]
+
+        raise RuntimeError(
+            "gfortran compiler not found. Please install it:\n"
+            "  macOS: brew install gcc\n"
+            "  Ubuntu/Debian: apt-get install gfortran\n"
+            "  RHEL/CentOS: yum install gcc-gfortran"
+        )
+
+    def initialize(self, version, build_data):
```

Ruff (0.14.4)

Unused method argument: `version`

(ARG002)